R/pkg/R/DataFrame.R

```diff
@@ -661,6 +661,34 @@ setMethod("saveAsParquetFile",
             write.parquet(x, path)
           })
 
+#' write.text
+#'
+#' Saves the content of the DataFrame in a text file at the specified path.
```

Comment:
might want to add the DataFrame must have a column with the name "value" I recall there is a similar doc clarification in Scala recently.